src/ios/Accessibility.ts

```diff
@@ -56,6 +58,13 @@ export class Accessibility extends NativeAccessibility {
         }
     }
 
+    private _trackQueueStatus(newState: string) {
+        if (this._isScreenReaderEnabled && ['background', 'inactive'].indexOf(newState)) {
```

Comment:
This will evaluate to false if the new state is 'background' because indexOf will return an index of 0 which is falsey. You need to add ">= 0" to the end of the statement.

Reply:
My bad - fixed